janitor/functions.py

```diff
@@ -4982,3 +4983,194 @@ def pivot_longer(
     )
 
     return df
+
+
+@pf.register_dataframe_method
+def pivot_wider(
+    df: pd.DataFrame,
+    index: Optional[Union[List, str]] = None,
+    names_from: Optional[Union[List, str]] = None,
+    values_from: Optional[Union[List, str]] = None,
+    flatten_levels: Optional[bool] = True,
+    values_from_first: Optional[bool] = True,
+    names_prefix: Optional[str] = None,
+    names_sep: Optional[str] = "_",
+    fill_value: Optional[Union[int, float, str]] = None,
+) -> pd.DataFrame:
+    """
+    Reshape data from long to wide form. The number of columns are
+    increased, while decreasing the number of rows. It is the inverse
+    of the `pivot_longer` method. It is a wrapper around `pd.unstack`
+    method.
+
+    This method does not mutate the original DataFrame.
+
+
+    Reshaping to wide form :
+
+    .. code-block:: python
+
+             name variable  value
+        0   Alice      wk1      5
+        1   Alice      wk2      9
+        2   Alice      wk3     20
+        3   Alice      wk4     22
+        4     Bob      wk1      7
+        5     Bob      wk2     11
+        6     Bob      wk3     17
+        7     Bob      wk4     33
+        8   Carla      wk1      6
+        9   Carla      wk2     13
+        10  Carla      wk3     39
+        11  Carla      wk4     40
+
+        df = pd.DataFrame(...).pivot_wider(index="name",
+                                           names_from="variable",
+                                           values_from="value")
```

Comment:
Let's follow `black` style here:

```suggestion
        df = (
            pd.DataFrame(...)
            .pivot_wider(
                index="name",
                names_from="variable",
                values_from="value"
            )
```